fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -1688,12 +1732,77 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 			writeThread = createGenericThreadPool();
 			readThreads = createGenericThreadPool();
 		}
+		commitLatencyHistogram = Histogram::getHistogram(ROCKSDBSTORAGE_HISTOGRAM_GROUP_ZHE, ROCKSDB_COMMIT_LATENCY_HISTOGRAM, Histogram::Unit::microseconds);
+		commitActionHistogram = Histogram::getHistogram(ROCKSDBSTORAGE_HISTOGRAM_GROUP_ZHE, ROCKSDB_COMMIT_ACTION_HISTOGRAM, Histogram::Unit::microseconds);
+		commitQueueWaitHistogram = Histogram::getHistogram(ROCKSDBSTORAGE_HISTOGRAM_GROUP_ZHE, ROCKSDB_COMMIT_QUEUEWAIT_HISTOGRAM, Histogram::Unit::microseconds);
+		writeHistogram = Histogram::getHistogram(ROCKSDBSTORAGE_HISTOGRAM_GROUP_ZHE, ROCKSDB_WRITE_HISTOGRAM, Histogram::Unit::microseconds);
+		deleteCompactRangeHistogram = Histogram::getHistogram(ROCKSDBSTORAGE_HISTOGRAM_GROUP_ZHE, ROCKSDB_DELETE_COMPACTRANGE_HISTOGRAM, Histogram::Unit::microseconds);
+		readRangeLatencyHistogram = Histogram::getHistogram(ROCKSDBSTORAGE_HISTOGRAM_GROUP_ZHE, ROCKSDB_READRANGE_LATENCY_HISTOGRAM, Histogram::Unit::microseconds);
+		readValueLatencyHistogram = Histogram::getHistogram(ROCKSDBSTORAGE_HISTOGRAM_GROUP_ZHE, ROCKSDB_READVALUE_LATENCY_HISTOGRAM, Histogram::Unit::microseconds);
+		readPrefixLatencyHistogram = Histogram::getHistogram(ROCKSDBSTORAGE_HISTOGRAM_GROUP_ZHE, ROCKSDB_READPREFIX_LATENCY_HISTOGRAM, Histogram::Unit::microseconds);
+		readRangeActionHistogram = Histogram::getHistogram(ROCKSDBSTORAGE_HISTOGRAM_GROUP_ZHE, ROCKSDB_READRANGE_ACTION_HISTOGRAM, Histogram::Unit::microseconds);
+		readValueActionHistogram = Histogram::getHistogram(ROCKSDBSTORAGE_HISTOGRAM_GROUP_ZHE, ROCKSDB_READVALUE_ACTION_HISTOGRAM, Histogram::Unit::microseconds);
+		readPrefixActionHistogram = Histogram::getHistogram(ROCKSDBSTORAGE_HISTOGRAM_GROUP_ZHE, ROCKSDB_READPREFIX_ACTION_HISTOGRAM, Histogram::Unit::microseconds);
+		readRangeQueueWaitHistogram = Histogram::getHistogram(ROCKSDBSTORAGE_HISTOGRAM_GROUP_ZHE, ROCKSDB_READRANGE_QUEUEWAIT_HISTOGRAM, Histogram::Unit::microseconds);
+		readValueQueueWaitHistogram = Histogram::getHistogram(ROCKSDBSTORAGE_HISTOGRAM_GROUP_ZHE, ROCKSDB_READVALUE_QUEUEWAIT_HISTOGRAM, Histogram::Unit::microseconds);
+		readPrefixQueueWaitHistogram = Histogram::getHistogram(ROCKSDBSTORAGE_HISTOGRAM_GROUP_ZHE, ROCKSDB_READPREFIX_QUEUEWAIT_HISTOGRAM, Histogram::Unit::microseconds);
+		readRangeNewIteratorHistogram = Histogram::getHistogram(ROCKSDBSTORAGE_HISTOGRAM_GROUP_ZHE, ROCKSDB_READRANGE_NEWITERATOR_HISTOGRAM, Histogram::Unit::microseconds);
+		readValueGetHistogram = Histogram::getHistogram(ROCKSDBSTORAGE_HISTOGRAM_GROUP_ZHE, ROCKSDB_READVALUE_GET_HISTOGRAM, Histogram::Unit::microseconds);
+		readPrefixGetHistogram = Histogram::getHistogram(ROCKSDBSTORAGE_HISTOGRAM_GROUP_ZHE, ROCKSDB_READPREFIX_GET_HISTOGRAM, Histogram::Unit::microseconds);
+
+		actors.push_back(updateHistogram(this, latencyMeasure));
```

Comment:
1. Could we move this getHistogram block inside the metrics actors? 
2. Is it possible to use a single actor to read all histograms (not required)?